### ISSUE TITLE
Uploads not uploading to site

### ISSUE THREAD

**Issue by justinelliotmeyers** [hotosm/OpenAerialMap#143]
I tried three times this weekend to upload a new map/ project. I renamed each one so it did not think it was a duplicate or an issue. If they would successfully upload, I will go back and delete the two duplicates. For the past three weeks, all my uploads are working fine, but this weekend they would not work. I did receive two emails stating I can check the progress, but the links in them lands me at the main map page, and doesn't show anything else (*I thought it would show progress/ review or whatever the interface is).

Can someone here help? @smit1678 or @dakotabenjamin 

Thanks,
Justin

**Comment by dakotabenjamin** [comment]
Justin- thanks for bringing this to our attention. Would you mind giving some more information for us to address the issue? Specifically, the links from the emails of the failed uploads contain IDs that can help us track down the logs.

**Comment by justinelliotmeyers** [comment]
@dakotabenjamin 
Your upload has been successfully submitted and is now being processed. You can check on the status of the upload at https://u1963947.ct.sendgrid.net/ls/click?upn=kj66w9JiszH8JuR6WnBTijT9uxgou2KgZAANoftqqlSwwEWI-2BI0ljUhmGQdN2cq-2BZ3lRjXvA7aq6jX3-2BTRrcqZ-2FkyNdK03az41MAFp-2FH-2BIU-3DvGdY_YqVc2Q6TaxfIVY64HpFtF3xc-2B-2BNGGJhLm-2BuUdXgnNdYQ9M0m-2Bd1BZo3CxcFq2GHh4AeXq4AbAmb41yuLbO9XDFrw4d5ebAEZeCu4Z7-2F0L0b7DS-2FYnpxOAbtBkUuWdEDFpwaO9HpDz45baywRZHjFhoY5W8fY5Vij3A2dK5O1EeSg9L0q5knGpb3cTRvmIaG6Le59ZDPICSN5KI2i3CXCHAy9ZjVs-2FieUKINTGskJVww-3D

Your upload has been successfully submitted and is now being processed. You can check on the status of the upload at https://u1963947.ct.sendgrid.net/ls/click?upn=kj66w9JiszH8JuR6WnBTijT9uxgou2KgZAANoftqqlSwwEWI-2BI0ljUhmGQdN2cq-2BubSv86u9VIfGEfeqrFg5ttMkAmysyiBXHllD2HJGd3Y-3D7bAS_YqVc2Q6TaxfIVY64HpFtF3xc-2B-2BNGGJhLm-2BuUdXgnNdarc-2FFjJ0XaQkAl4EqlXrpg4cVBdNMW3AEIuNmGOyPlq0ty0l6Pxt-2BU-2FhM0HlXUzBo-2Fdi0MH-2FDwa0-2FP6cXrDLw9f6-2FA-2FF1-2FcEv6z6k-2FNUCkhZAnRvEe0Fn3BsS11nj6TELRxS6KrVXG94TTgfygFeoX1SvoaooJ-2FeUEjn99LcNgspgQg-2BYR0zHut1z6tt-2BlEek-3D

**Comment by p-phung** [comment]
It's my first time to contribute to OAM. And I have the same issue in December.

I used an imagery public link and I also tried to upload the image from my local computer but I only received an email about processing as above. Is this issued solved?

I thought that I might haven't done it properly, then I tried to follow [the documentation](https://docs.openaerialmap.org/uploader/uploader-form/). But the token request button doesn't work either.
